main.go

```diff
@@ -278,8 +278,8 @@ func (cmd *InstallCmd) Run(ctx *Context) error {
 
 					fmt.Printf("  Installing %s on Compute Node %s\n", d.Name, compute)
 
-					fmt.Printf("  Stopping service...")
-					cmd := exec.Command("ssh", compute, "systemctl", "stop", d.Bin, "|| true")
+					fmt.Printf("  Removing %s service...", d.Name)
+					cmd := exec.Command("ssh", compute, "/usr/bin/"+d.Bin, "remove", "|| true")
```

Comment:
Remove does a 'systemctl disable', but it doesn't do a stop.  You still need the stop.